scripts/export_database.pl

```diff
@@ -274,6 +278,7 @@ sub sanitize_field_content {
 		get_products_collection({obsolete => 1, timeout => 3 * 60 * 60 * 1000})
 	);
 
+	push @export_fields, "obsolete";
```

Comment:
This is not the right place to add it.

Add it to Config_off.pm (and obf/opff/opf for provision).